src/main/java/ai/tecton/client/model/FeatureValue.java

```diff
@@ -159,7 +165,19 @@ public Value(Object featureObject, ValueType valueType) {
           break;
         case FLOAT64:
           // Tecton also represents all double feature values as JSON numbers in the response.
-          this.float64Value = (Double) featureObject;
+          if (featureObject instanceof String) {
+            String doubleString = (String) featureObject;
+            if (doubleString.equals("null")
```

Comment:
Does the service return `"null"` as a string? Wouldn't it return it as json-null instead?

Reply:
I believe we already handle json-nulls separately

Reply:
Yes, so wondering if there are cases where the service return `"null"` as a string instead of json-null

Reply:
I don't believe so...its probably a defensive change

Reply:
Accurate, more defensive.